src/test/scala/io/iohk/ethereum/mpt/MerklePatriciaTrieSuite.scala

```diff
@@ -437,4 +417,4 @@ class MerklePatriciaTrieSuite extends FunSuite
     val storage = EmptyTrie.put(key1, key1).put(key2, key2).put(key3, key3)
     assert(Hex.toHexString(storage.getRootHash) == "a3d0686205c7ed10a85c3bce4118d5d559bcda47ca39e4dd4f09719958a179f1")
   }
-}
\ No newline at end of file
+}
```

Comment:
@mcsherrylabs I understood we will create a new task for this as Iodb doesn't have either, do you want me to include those tests in this PR?